### DISCUSSION TITLE
Search Count in Lualine

### DISCUSSION THREAD

**Discussion by kcaashish** [LazyVim/LazyVim#848] [Q&A]
How and where is this search-counter implemented? 
<img width="466" alt="image" src="https://github.com/LazyVim/LazyVim/assets/31175326/1f1fcc6e-0a39-4395-beb1-ba19c78c1f8a">

This doesn't display when the jump to next search result is large. So, I would like to have this in the lualine. How can I do that?

**Comment by jyuan0** [comment]
This is implemented in [Noice](https://github.com/folke/noice.nvim) as one of the default routes. You can add it to lualine in an `opts` function with something like this:
```
  {
    "nvim-lualine/lualine.nvim",
    opts = function(_, opts)
      -- stylua: ignore
      table.insert(opts.sections.lualine_x, {
        function() return require("noice").api.status.search.get() end,
        cond = function() return package.loaded["noice"] and require("noice").api.status.search.has() end,
        color = require("lazyvim.util").fg("Statement"),
      })
    end,
  }
```
